freeipa/src/main/resources/freeipa-salt/salt/upgradeccm/schedule.sls

```diff
@@ -0,0 +1,14 @@
+#!py
+import datetime
+
+def run():
+    date = datetime.datetime.now() + datetime.timedelta(minutes=__pillar__['ccm_jumpgate']['activation_in_minutes'])
+    return {"restore_job": {
+    "schedule.present": [
+       {"function": "state.sls"},
+       {"job_args": ["upgradeccm.restoressl"]},
+       {"once": date.strftime("%Y/%m/%d, %H:%M:%S")},
+       {"once_fmt": "%Y/%m/%d, %H:%M:%S"}
+     ]
+  }}
```

Comment:
wow, can't be this solved with simple jinja? like mentioned here: https://groups.google.com/g/salt-users/c/TqV2PlSKDkA

Reply:
what I can't find is a simple way to do the math in jinja. It is straightforward now